src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/controllers/StaleBlobControllerTest.java

```diff
@@ -41,8 +42,19 @@ void should_return_list_of_stale_blobs_when_there_is_with_request_param() throws
 
         given(staleBlobFinder.findStaleBlobs(60))
             .willReturn(Arrays.asList(
-                new BlobInfo("container1", "file_name_1", createdAt),
-                new BlobInfo("container2", "file_name_2", createdAt))
+                new BlobInfo(
+                    "container1",
+                    "file_name_1",
+                             UUID.fromString("690d4100-7ffe-11eb-9439-0242ac130002"),
```

Comment:
It should be:
`var id1 = UUID.randomUUID();`
`var id2 = UUID.randomUUID();`
and then use `id1`, `id2` instead of
`UUID.fromString("690d4100-7ffe-11eb-9439-0242ac130002")` and `UUID.fromString("77cf250a-7ffe-11eb-9439-0242ac130002")`

Reply:
I see. You are right it adds readability.

Reply:
Changes made.